base/strings/util.jl

```diff
@@ -427,16 +497,19 @@ function replace(str::String, pat_repl::Pair; count::Integer=typemax(Int))
 end
 
 """
-    replace(s::AbstractString, pat=>r; [count::Integer])
+    replace(s::AbstractString, pat=>r[, pat=>r ...]; [count::Integer])
```

Comment:
Just add `...` after the first `pat => r`, no need to repeat it.

Reply:
done.